modules/distribution-zones/build.gradle

```diff
@@ -55,6 +55,7 @@ dependencies {
     testImplementation project(':ignite-metastorage')
     testImplementation project(':ignite-system-view-api')
     testImplementation project(':ignite-configuration-system')
+    testImplementation project(':ignite-replicator')
```

Comment:
Distribution zones should not depend on replicator.

Reply:
🆗